spec/swagger.yaml

```diff
@@ -236,6 +240,73 @@ paths:
             -d '{"refreshToken": "eyJhbGciO..."}'
 
 #
+# Accounts & Projects
+#
+
+  '/accounts-api/v2/accounts/{accountUid}/projects':
+    get:
+      summary: List Projects
+      description: |
+        Return the list of projects for the account
```

Comment:
Returns the list of projects for the account.

Reply:
@hchaps Not related to this particular PR, but wanted to point out to other inconsistencies in use of third form singular form of verbs in our api docs. For example `Return information on a specific file` https://smartling.github.io/api-docs/#tag/Files%2Fpaths%2F~1files-api~1v2~1projects~1%7BprojectId%7D~1file~1status%2Fget.

Reply:
applied change